cnxeasybake/oven.py

```diff
@@ -386,7 +402,7 @@ def build_recipe(self, element, step, depth=0):
             if 'outside_deferred' in matching_rules:
                 for rule, declarations in \
                         matching_rules.get('outside_deferred'):
-                    logger.debug(u'Rule ({}): {}'.format(*rule))
+                    self.report_coverage(rule)
```

Comment:
Hmm, this repetition seems repetitive. Wonder if we can piggyback on the debug decorator (or make such a decorator)